Assets/scripts/UI/View/ViewStartUI.cs

```diff
@@ -8,15 +9,22 @@ public class ViewStartUI : MonoBehaviour
 {
     Text score;
     Text health;
+    Text achievement;
     public void Start()
     {
+        if (ServiceAchievements.Instance != null)
```

Comment:
was this throwing a NPE ?

Reply:
I did it for a case if I disable achievements.